lib/pulsar/producer.rb

```diff
@@ -22,9 +22,9 @@
 module Pulsar
   class Producer
     module RubySideTweaks
-      def send(message)
+      def send(message, partitionKey='')
```

Comment:
while `""` rather than `nil` is what's needed for the CPP binding, using or at least allowing nil would be a lot more idiomatic for ruby. maybe default to nil, and then default nil -> "" on line 27 (e.g. `new(message, partitionKey || '')`)

Reply:
ack, I agree